src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/ManifestTest.cs

```diff
@@ -97,6 +97,7 @@ public void Bug12935 ()
 				builder.Cleanup ();
 				proj.TargetFrameworkVersion = "v4.1";
 				proj.AndroidManifest = string.Format (TargetSdkManifest, "16");
+				proj.OtherBuildItems [0].Timestamp = null;
```

Comment:
we should probably put this code in the `project.AndroidManifest` property setter (or something like it) so that if the property is updated it will do this.